roles/docker/tasks/set_facts_dns.yml

```diff
@@ -45,8 +45,7 @@
 
 - name: add system search domains to docker options
   set_fact:
-    docker_dns_search_domains: "{{ docker_dns_search_domains + [item] }}"
-  with_items: "{{ system_search_domains.stdout.split(' ') }}"
+    docker_dns_search_domains: "{{ docker_dns_search_domains + system_search_domains.stdout.split(' ')|default([]) }}|unique"
```

Comment:
I'm a bit worried about unique here as it may change order of the entries.Not sure if this is important for search domains.